packages/elements-react/src/components/form/nodes/input.tsx

```diff
@@ -37,13 +37,13 @@ export const NodeInput = ({
 
   const setFormValue = () => {
     if (
-      isResendNode ||
-      isScreenSelectionNode ||
-      node.group === UiNodeGroupEnum.Oauth2Consent
+      attrs.value &&
```

Comment:
I don't understand why the other logic broke it, I really don't.

Reply:
@jonas-jonas do you understand this?

Reply:
Probably a re-rendering issue.. I didn't look into it too closely, https://github.com/ory/elements/issues/434 seems similar.